### PR TITLE
fix for non json events

### PR DESCRIPTION
**Pull request by Adamowoc**
Fix for AWS non json incoming events 
> sekoia_automation.module - CRITICAL - unexpected character: line 1 column 1 (char 0)

### CODE REVIEW

**Review by PierrickV — COMMENTED** [review]
Codecov reported an issue on the code that you added. I wonder if your test triggers the issue as excepted

File: AWS/aws/s3/queued.py [hunk 1]
```diff
@@ -79,14 +79,18 @@ def get_next_messages(self, **kwargs) -> list[dict]:
...
+                for record in decoded_messages:
+                    try:
+                        yield AWSS3Object.from_record(self.s3_client, record)
+
+                        if receipt := message.get("ReceiptHandle"):
+                            receipts.append(receipt)
+                    except ValueError as e:
+                        self.log_exception(e, message="Invalid record")
+            except ValueError as e:
+                self.log_exception(e, message=f"Invalid JSON in message.\nInvalid message is: {message}")
```
Comment (gaelmuller) [inline AWS/aws/s3/queued.py:93]: The except should probably be at line 84 since it is only covering the json deserialization (same for other instances)

**Review by gaelmuller — APPROVED** [review]

**Review by PierrickV — COMMENTED** [review]
You still have the issue where the SQS code does not seem tested
image

**Comment by PierrickV** [comment]
The error despaired when I reloaded :shrug: 

This other error is not tested btw https://app.codecov.io/gh/SEKOIA-IO/automation-library/pull/94/blob/AWS/aws/s3/queued.py#L94

**Review by PierrickV — APPROVED** [review]
Seems good to me

It would be nice to improve the coverage of this code :smile:

**Pull request merged**